src/aiofiles/base.py

```diff
@@ -1,16 +1,13 @@
-from asyncio import get_running_loop
+from asyncio import get_running_loop, to_thread
 from collections.abc import Awaitable
 from contextlib import AbstractAsyncContextManager
-from functools import partial, wraps
+from functools import wraps
 
 
 def wrap(func):
```

Comment:
@Tinche , would you like to rename this function into something like "threadify"? Not now and not harshly, but with an alias function and a deprecation warning. Sounds good? If so, what name would you choose?